packages/hardhat/hardhat.config.ts

```diff
@@ -10,6 +17,40 @@ const config: HardhatUserConfig = {
       default: 0,
     },
   },
+  networks: {
+    arbiturum: {
```

Comment:
There is a typo here. "arbiturum" => "arbitrum"

Reply:
changed here https://github.com/scaffold-eth/se-2/pull/24/commits/edeadba1050c1bbf017050d127156399364fe3e4